docs/getting-started/post-installation/atlas-folder/interface-tweaks.md

```diff
@@ -25,7 +25,13 @@ Although such gestures are meant to be convenient, many users are inconvenienced
 
 ## :material-menu-open: Context Menus
 
-### Run With Priority in the context menu
+### Extract in the Context Menu
+
+By default, Atlas hides the extract option in the context menu and replaces it with [NanaZip](https://github.com/M2Team/NanaZip).
+
+To revert this, run `Add Extract.reg`, go up a directory and run `Restart Explorer.cmd`.
```

Comment:
Instead of saying 'To revert this,' consider saying 'To add back the extract context menus' because it wouldn't uninstall NanaZip